exporter/dorisexporter/factory.go

```diff
@@ -0,0 +1,54 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package dorisexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/dorisexporter"
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configretry"
+	"go.opentelemetry.io/collector/exporter"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/dorisexporter/internal/metadata"
+)
+
+func NewFactory() exporter.Factory {
+	return exporter.NewFactory(
+		metadata.Type,
+		createDefaultConfig,
+		exporter.WithLogs(createLogsExporter, metadata.LogsStability),
+		exporter.WithTraces(createTracesExporter, metadata.TracesStability),
+		exporter.WithMetrics(createMetricsExporter, metadata.MetricsStability),
+	)
+}
+
+func createDefaultConfig() component.Config {
+	return &Config{
+		TimeoutSettings: exporterhelper.NewDefaultTimeoutSettings(),
+		QueueSettings:   exporterhelper.NewDefaultQueueSettings(),
+		BackOffConfig:   configretry.NewDefaultBackOffConfig(),
+		Table: Table{
+			Logs:    "otel_logs",
+			Traces:  "otel_traces",
+			Metrics: "otel_metrics",
+		},
+		Database:     "otel",
+		CreateSchema: true,
+		HistoryDays:  0,
+		TimeZone:     "Asia/Shanghai",
```

Comment:
The default `TimeZone`, I think it's best to be the time zone of the server where the collector is located

Reply:
Ok, I'll fix it.